spec/lib/runner_spec.rb

```diff
@@ -362,6 +362,16 @@ def handle_valid_record(record, data_type)
         expect(@handler.records_seen['simple-licence']).to eq(10)
       end
     end
+
+    context 'when script output files are missing' do
+      it 'proceeds without errors' do
+        runner = test_runner('bot-with-transformer',
+          :record_handler => @handler
+        )
+        FileUtils.rm_rf(File.join(runner.base_directory, "output"))
```

Comment:
Yes at the moment this test is ensuring the files are non-existant. We could make sure all the other tests are undoing their file changes in an "after" block. Would that be best practice? or we could do _both_ perhaps.

Reply:
OK. I've added lots of "after" blocks because each test should clean up any files it creates. This is repetitive, and it's tempting to reduce repetition by assuming the setting of `@runner`, but maybe that would be a big global-variable-ish